src/buildercore/trop.py

```diff
@@ -360,6 +360,16 @@ def render_s3(context, template):
             'DeletionPolicy': context['s3'][bucket_name]['deletion-policy'].capitalize()
         }
         bucket_title = _sanitize_title(bucket_name) + "Bucket"
+        if context['s3'][bucket_name]['cors']:
+            props['CorsConfiguration'] = s3.CorsConfiguration(
+                CorsRules=[
+                    s3.CorsRules(
+                        AllowedHeaders=['*'],
+                        AllowedMethods=['GET', 'HEAD'],
+                        AllowedOrigins=['*']
```

Comment:
maybe if it were within an intranet or secured or something. I like the idea of POSTing something to a bucket and then having the change to a bucket's contents trigger other changes, reactive style